thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/StaticPipeline.java

```diff
@@ -0,0 +1,29 @@
+package com.linkedin.thirdeye.rootcause;
+
+import java.util.Set;
+
+
+/**
+ * StaticPipeline emits a fixed set of entities as a result. It is typically used to encapsulate
```

Comment:
removed "typically". It is used to encapsulate user input during framework execution.